pkg/push/api/api.go

```diff
@@ -720,17 +864,36 @@ func (s *PushService) DeletePush(
 		if err != nil {
 			return err
 		}
-		handler, err := command.NewPushCommandHandler(editor, push, s.publisher, req.EnvironmentNamespace)
+		prev := &domain.Push{}
+		if err = copier.Copy(prev, push); err != nil {
+			return err
+		}
+		push.Deleted = true
+		event, err = domainevent.NewEvent(
+			editor,
+			eventproto.Event_PUSH,
+			push.Id,
+			eventproto.Event_PUSH_CREATED,
```

Comment:
This event type should be `PUSH_DELETED`

Reply:
Thank you for pointing out! I have updated.